.github/workflows/psammead-chromaticqa.yml

```diff
@@ -17,10 +17,11 @@ jobs:
         node-version: ['12']
     env:
       CI: true
-      CHROMATIC_APP_CODE: ${{ secrets.CHROMATIC_APP_CODE }}
```

Comment:
Are these changes expected @ryanmccombe?

Reply:
aye - Chris added these changes to fix an issue with Chromatic failing on CI